ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -304,12 +302,11 @@ StatusModal {
 
     RenameGroupPopup {
         id: renameGroupPopup
-        // Not Refactored Yet
-//        activeChannelName: popup.store.chatsModelInst.channelView.activeChannel.name
-//        onDoRename: {
-//            popup.store.chatsModelInst.groups.rename(groupName)
-//            popup.header.title = groupName
-//            close()
-//        }
+        activeChannelName: popup.store.chatsModelInst.channelView.activeChannel.name
+        onDoRename: {
+            popup.store.chatsModelInst.renameGroup(groupName)
```

Comment:
It's weird that almost all group functions come from the `chatSectionModule`, but this one is on the `chatsModelInst`. Are you sure this is not an error and it should be `chatSectionModule`?